### PR TITLE
Add Product and Orders

### PR DESCRIPTION
**Pull request by MesropMkhitaryan**

### CODE REVIEW

File: src/main/java/com/mulberry/blackmulberry/controller/main/AnyController.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 public class AnyController {
```
Comment (Mushegh128) [inline src/main/java/com/mulberry/blackmulberry/controller/main/AnyController.java]: i believe that its not good name for controller. please change it. you can rename for excample Main or Home.

File: src/main/java/com/mulberry/blackmulberry/entity/CategoryProduct.java [hunk 2]
```diff
@@ -20,5 +20,6 @@ public class CategoryProduct {
     private int id;
     private String name;
     @ManyToOne
+    @JoinColumn(name="cat_par_id")
```
Comment (Mushegh128) [inline src/main/java/com/mulberry/blackmulberry/entity/CategoryProduct.java]: i know that cat its animal. lets change column name to category_parent_id.

File: src/main/java/com/mulberry/blackmulberry/entity/Product.java [hunk 3]
```diff
@@ -24,6 +24,12 @@ public class Product {
     private String picUrl;
     private Date createAt;
     private String description;
-
+    @ManyToOne
+    @JoinColumn(name="cat_id")
```
Comment (Mushegh128) [inline src/main/java/com/mulberry/blackmulberry/entity/Product.java]: here also is sweety animal. its not cat_id

**Comment by MesropMkhitaryan** [comment]
I resolved all the issues that we had with our project

**Review by Mushegh128 — APPROVED** [review]

**Pull request merged**